hydra-cluster/test/Test/DirectChainSpec.hs

```diff
@@ -139,13 +135,12 @@ spec = around showLogsOnFailure $ do
     alicesCallback <- newEmptyMVar
     withTempDir "hydra-cluster" $ \tmp -> do
       config <- newNodeConfig tmp
-      aliceKeys@(aliceCardanoVk, aliceCardanoSk) <- keysFor Alice
-      withBFTNode (contramap FromCluster tracer) config [aliceCardanoVk] $ \node@(RunningNode _ nodeSocket) -> do
-        pparams <- queryProtocolParameters defaultNetworkId nodeSocket
+      aliceKeys@(aliceCardanoVk, _) <- keysFor Alice
+      withBFTNode (contramap FromCluster tracer) config $ \node@(RunningNode _ nodeSocket) -> do
         let cardanoKeys = [aliceCardanoVk]
         withIOManager $ \iocp -> do
           withDirectChain (contramap (FromDirectChain "alice") tracer) magic iocp nodeSocket aliceKeys alice cardanoKeys (putMVar alicesCallback) $ \Chain{postTx} -> do
-            postSeedPayment (Testnet magic) pparams availableInitialFunds nodeSocket aliceCardanoSk 100_000_000
+            void $ seedFromFaucet (Testnet magic) node aliceCardanoVk 100_000_000 Marked
```

Comment:
Proposal: Rather than exposing the `seedFromFaucet` function with an additional parameter, expose 2 different functions with explicit names, eg. `createFuelUTxO` and `createCommitUTxO` or something. Additional parameters controlling the branching of a function increases the cognitive load of the function's user or reader

Reply:
I'm not sure about this. The parameter does not control the branching of this function, rather what the thing you get from the faucet will be. Similar to the amount or whom it is payed to. I admit that the naming is not good and maybe `data Marked = Fuel | NotMarked` would be a better name?